.travis.yml

```diff
@@ -16,9 +17,11 @@ before_install:
 # If we are building as gonum, then we have access to the coveralls api key, so we can run coverage as well.
 script:
  - go get -d -t -v ./...
- - go build -v ./...
- - go test -v -a ./...
- - go test -v -a -tags noasm ./...
- - go test -v -a -tags appengine ./...
+ - go build -v -x ./...
+ - go test -v -x -a ./...
+ - go test -v -x -a -tags noasm ./...
+ - go test -v -x -a -tags appengine ./...
  - diff <(gofmt -d .) <("")
```

Comment:
Good idea but perhaps we should wait until it stabilizes a bit? I will add a TODO.